version

```diff
@@ -1,5 +1,5 @@
 
 # Copyright (C) NGINX, Inc.
 
-NXT_VERSION=1.32.0
-NXT_VERNUM=13200
+NXT_VERSION=1.32.1
+NXT_VERNUM=13201
```

Comment:
The current version is **1.33.0**

Reply:
I don't know where it came from, but it's not part of this PR